checks/macos.py

```diff
@@ -66,3 +66,28 @@ def checkRosettaTranslationStatus(lines):
     if (len(search('Rosetta translation used: true', lines)) > 0):
         return [LEVEL_WARNING, "Intel OBS on Apple Silicon Mac",
                 "You are running the Intel version of OBS on an Apple Silicon Mac. You may get improved performance using the Apple Silicon version of OBS."]
+
+
+def checkMacPermissions(lines):
+    macPerms = search('[macOS] Permission for', lines)
+    deniedPermissions = []
+
+    for line in macPerms:
+        if 'denied' in line:
+            found = re.search(r'Permission for (.+) denied', line)
+            if found:
+                permissionName = found.group(1).title()
+                permissionDescription = {
+                    "Audio Device Access": "<b>Microphone</b>: OBS requires this permission if you want to capture your microphone.",
+                    "Video Device Access": "<b>Camera</b>: This permission is needed in order to capture content from a webcam or capture card.",
+                    "Accessibility": "<b>Accessibility</b>: For keyboard shortcuts (hotkeys) to work while other apps are focused this permission is needed.",
+                    "Screen Capture": "<b>Screen Recording</b>: OBS requires this permission to be able to capture your screen."
+                }.get(permissionName)
+
+                if permissionDescription:
+                    deniedPermissions.append(permissionDescription)
+
+    if deniedPermissions:
+        deniedPermissionsString = "<br><ul><li>" + "</li><li>".join(deniedPermissions) + "</li></ul>"
```

Comment:
Add line breaks appropriately or the console output becomes unreadable.

Reply:
Please apply newlines as needed. We'll figure out how to get the output formatting (textwrap.fill) to behave another time, but at least the strings will be ready.

Reply:
Updated with new lines added.